test/parser/comments.test.js

```diff
@@ -28,14 +28,45 @@ test('inline comment without leading space', (t) => {
 });
 
 test('inline comment with unclosed characters', (t) => {
-  const less = `// unclosed ' test { test ( test /*`;
+  const less = `// unclosed ' test { test ( test /* test "\nfoo: 'bar';\nbar: "foo"\nabc: (def)\n foo{}`;
   const root = parse(less);
   const { first } = root;
 
   t.truthy(root);
   t.true(first instanceof Comment);
   t.true(first.inline);
-  t.is(first.text, less.slice(3));
+  // first.text should only contain the comment line without the `// `
+  t.is(first.text, less.substring(0, less.indexOf('\n')).slice(3));
```

Comment:
please break this out into a different test. I feel that the changes to the existing test are going after a different result than the original test was meant for.

Reply:
Done